app/models/group_offer.rb

```diff
@@ -0,0 +1,41 @@
+class GroupOffer < ApplicationRecord
+  belongs_to :department, optional: true
+
```

Comment:
why isnt there a relation to volunteer?

Reply:
Because that is part of a [following story](https://trello.com/c/MGYQ5fwy/4-5-als-freiwilligenverantwortliche-r-m%C3%B6chte-ich-ein-oder-mehrere-freiwillige-mit-gruppenangeboten-matchen).